beets/dbcore/db.py

```diff
@@ -842,11 +882,15 @@ def _fetch(self, model_cls, query=None, sort=None):
         """
         query = query or TrueQuery()  # A null query.
         sort = sort or NullSort()  # Unsorted.
-        where, subvals = query.clause()
+        where, subvals, new_tables = query.clause()
```

Comment:
@sampsyo should I still allow Query objects to maybe return 2-tuples just in case any external plugin still uses the old API?  It's easy to check for unpack errors but it's a bit ugly, so I preferred to ask your opinion.